<launch>
    <arg name="visualization" default="true"/>

    <node pkg="tf" type="static_transform_publisher" name="world_map" args="0 0 0 0 0 0 world map 10"/>

    <node pkg="terrain_planner" type="test_rrt_circle_goal" name="rrt_planner" output="screen">
        <param name="map_path" value="$(find terrain_models)/models/sertig.tif"/>
        <param name="color_file_path" value="$(find terrain_models)/models/sertig_color.tif"/>
    </node>
    <group if="$(arg visualization)">
        <node type="rviz" name="rviz" pkg="rviz" args="-d $(find terrain_planner)/launch/config_ompl.rviz" />
    </group>
</launch>
